test/utils/MultipartFormDataWritable.scala

```diff
@@ -57,10 +55,6 @@ object MultipartFormDataWritable {
   val singleton = Writeable[MultipartFormData[TemporaryFile]](
     transform = { form: MultipartFormData[TemporaryFile] =>
       formatDataParts(form.dataParts) ++
-        form.files.flatMap { file =>
```

Comment:
Do you think we can get rid of this whole object? I can't find anywhere it (or the implicit) is in use.

Reply:
Good spot, doesn't look like its being used. I've removed in the recent commit.

Reply:
I just checked the new commit, I think we can get rid of the whole file.